templates/shared/actions/runner_list.tmpl

```diff
@@ -26,7 +26,7 @@
 					</div>
 					<div class="divider"></div>
 					<div class="item">
-						<a href="{{if $.isFromOrg}}{{$.Link}}/reset_registration_token{{else}}{{$.Link}}/runners/reset_registration_token{{end}}">Reset registration token</a>
+						<a href="{{if $.PageIsOrgSettings}}{{$.Link}}/reset_registration_token{{else}}{{$.Link}}/runners/reset_registration_token{{end}}">Reset registration token</a>
```

Comment:
Well, I believe we should use a simple line here:

```
						<a href="{{.LinkUrlResetRegistrationToken}}">Reset registration token</a>
```

Just let callers prepare it.

It's really fragile to make sub-templates "guess" which parent template they are in.

Reply:
Added a commit https://github.com/go-gitea/gitea/pull/24200/commits/7a968add85befcb56036cf0ce8e6b1459c9c60ec to change the data passed to the templates